testdata/p4_14_samples_outputs/copy_to_cpu.p4

```diff
@@ -3,7 +3,8 @@
 #include <v1model.p4>
 
 enum bit<8> FieldLists {
-    copy_to_cpu_fields = 0
+    none = 0,
```

Comment:
Is adding this `none` field intentional?

Reply:
Thank you Andy!

Reply:
Yes, that is correct, just to make sure that id 0 is not used by other field lists we allocate it to none and no fields will be tagged with none. So calling recirculate() will not recirculate any fields.